src/tools/picker-selection.ts

```diff
@@ -0,0 +1,37 @@
+import { Events } from "../events";
+
+class PickerSelection {
+    events: Events;
+    root: HTMLElement;
+
+    constructor(events: Events, parent: HTMLElement) {
+        this.root = document.createElement('div');
+        this.root.id = 'select-root';
+
+        this.root.onmousedown = (e: MouseEvent) => {
```

Comment:
Can we aim to stick to pointer events in our apps from now on?

Reply:
Yes! I will update everything to pointer events in a separate PR.